File: Projekat_WEB2_backend/Projekat_WEB2_backend/Dto/PorudzbinaDto.cs
using Projekat_WEB2_backend.Enumerations;
using Projekat_WEB2_backend.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Projekat_WEB2_backend.Dto
{
    public class PorudzbinaDto
    {
        public long Id { get; set; }
        public string Komentar { get; set; }
        public string Adresa { get; set; }
        public DateTime DatumDostave { get; set; }
        public DateTime DatumKreiranja { get; set; }
        public StanjePorudzbine StanjePorudzbine { get; set; }
        public long KorisnikId { get; set; }
        public List<ArtikalPorudzbineDto> ArtikliPorudzbine { get; set; }
        public double CenaPorudzbine { get; set; }
       

        
    }
}

File: projekat_web2_frontend/src/Models/PorudzbinaDto.js
import ArtikalPorudzbineDto from '../Models/ArtikalPorudzbineDto';

export default class PorudzbinaDto{
    constructor(data){
        this.id = data.id;
        this.komentar = data.komentar;
        this.adresa = data.adresa;
        this.datumDostave = data.datumDostave;
        this.datumKreiranja = data.datumKreiranja;
        this.stanjePorudzbine = data.stanjePorudzbine;
        this.korisnikId = data.korisnikId;
        this.artikliPorudzbine = [];
        this.cenaPorudzbine = data.cenaPorudzbine;
    }


    addAllArtiklePorudzbine(artikliPorudzbine){
        for(var i = 0; i < artikliPorudzbine.length; i++){
            this.artikliPorudzbine.push(artikliPorudzbine[i]);
        }
    }
}